### DISCUSSION TITLE
アイテム仕様まとめ

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#100] [各種システムまとめ]
主にアイテムの仕様に関するまとめ
(システムに関係することはアイテムシステムに書く)

**Comment by anminmakura** [comment]
## バニラレシピで無効予定のものまとめ

### ・レッドストーン以外の宝石系ブロック→宝石、宝石→宝石ブロック系を全て無効
元素材 |   | 変換素材
-- | -- | --
ラピスラズリーブロック | → | ラピスラズリー
エメラルドブロック | → | エメラルド
ダイヤモンドブロック | → | ダイヤモンド
ネザライトブロック | → | ネザライト
ラピスラズリー | → | ラピスラズリーブロック
エメラルド | → | エメラルドブロック
ダイヤモンド | → | ダイヤモンドブロック
ネザライト | → | ネザライトブロック

**2022/6/14 現在においては宝石系ブロックのレシピは無効にしなくても進行に問題がないことがわかったので
この案は没。**

### ・エリトラの修繕を無効
代わりに12vと比較して安価で買えるようにし、エリトラは耐久度は通常の1/5の予定。
13vの剣士のスキル「鍛治」を使用すると耐久値が増えるのでそれの調整も兼ねている。

・花火は無効、報酬として入れるのも規制
エリトラのぶっ壊れ強化アイテムである花火は入手不可、製作不可とする。

### ・ビーコンを無効
一見ネザースターが経験値として使用しており対策は万全のように見えるが、
ネザースター以外の素材を並べた状態で待機し、
収納したネザースターをドロッパーを用いる事でコンベア式でプレイヤーに渡し、
インベントリー移動を連打することで抜けることが可能だったので、今回は完全に封殺予定。

**Comment by anminmakura** [comment]
## 改造レシピで入れるものまとめ

### ・上位金リンゴを超作業台で作りたい
1.16のマイクラから上位金リンゴが仕様変更により大幅弱体化するので、
必然的に価値が下がるので作れるようにはする。

### ・ガラス瓶→ガラス
あると便利だと思うので製作予定。

### ・棒と木の感圧版で額縁
v12では+木のボタンだったが不要と判断

**Comment by anminmakura** [comment]
- [ ] ポーションのガソリンをトカルトの採掘場とかで売る。

**Comment by Suyarumi** [comment]
### アイテム作成時のルール
***
アイテム名、説明文はjsonで統一する。
Attribute UUIDはスロットによって決まっている。
例:
mainhand: UUID:[I; 0,0,0,1]
offhand: UUID:[I; 0,0,0,2]
